tests/e2e/minikube-kubernetes-istio/OWNERS

```diff
@@ -0,0 +1,4 @@
+gargnupur
```

Comment:
I suspect we can use simpler directory name. =)

perhaps something like
test/e2e/local/vagrant
test/e2e/local/minikube

Reply:
Thanks for the advice, done.